README.md

```diff
@@ -60,20 +67,20 @@ provider:
 1. Install Ironic. There are a couple of ways to do it.
     - Run within a Kubernetes cluster as a pod, refer to [deploy.sh](https://github.com/metal3-io/baremetal-operator/blob/main/tools/deploy.sh)
       script.
-    - Outside of a Kubernetes cluster as a container. Please refer to [run_local_ironic.sh](https://github.com/metal3-io/baremetal-operator/blob/main/tools/run_local_ironic.sh).
+    - Outside of a Kubernetes cluster as a container. Please refer to [run_local_ironic.sh](https://github.com/metal3-io/baremetal-operator/blob/main/tools/run_local_ironic.sh) script.
 
 Please refer to the [getting-started](docs/getting-started.md) for more info.
 
 ## Pivoting ⚠️
 
-Starting from `v0.5.0` release of Cluster-api-provider-metal3, Baremetal Operator is decoupled
-from Cluster-api-provider-metal3 deployments when deployed via `clusterctl`. For that reason,
+Starting from `v0.5.0` release of Cluster API Provider Metal3, Baremetal Operator is decoupled
+from Cluster API Provider Metal3 deployments when deployed via `clusterctl`. For that reason,
 when performing `clusterctl move`, custom objects outside of the Cluster API chain or not part
-of CAPM3 will not be pivoted to a target cluster. Example to those objects is BareMetalHost, its
+of CAPM3 will not be pivoted to a target cluster. An example of those objects is BareMetalHost, it's
```

Comment:
I think it should be "its" here, but maybe we could change it a bit to clarify this is not just about BMO resources?
```suggestion
of CAPM3 will not be pivoted to a target cluster. An example of those objects is BareMetalHost, or user created ConfigMaps and Secrets.
```

Reply:
Thanks, fixed.